client/log_verifier.go

```diff
@@ -56,13 +59,19 @@ func NewLogVerifierFromTree(config *trillian.Tree) (*LogVerifier, error) {
 		return nil, fmt.Errorf("client: NewLogVerifierFromTree(): NewLogHasher(): %v", err)
 	}
 
-	// Log Key
+	// Log Pub Key
```

Comment:
(nit) no need to abbreviate in a comment

Reply:
done